src/frontend/App/typer/toast.ts

```diff
@@ -21,4 +22,6 @@ export const toastTilTekst: Record<EToast, string> = {
     VEDTAK_NULLSTILT: 'Lagret vedtak nullstilt',
     OPPRETTET_UTESTENGELSE: 'Utestengelse satt på bruker',
     BREV_SENDT: 'Brevet er nå sendt',
+    REDIRECT_ANNEN_RELASJON_FEILET:
+        'Kan ikke åpne personopplysninger for denne personen. En uventet feil oppstod',
```

Comment:
Tar gjerne feedback på en god feilmelding dersom vi av en eller annen grunn ikke lykkes i å lage en fagsakperson... Fant ikke noe bedre "visning" enn toast på dette - da det blir veldig snodig å sprette opp en rød klump med feilmelding i de forskjellige små områdene selve lenkene befinner seg...

Reply:
Går det an å tilby en feilmelding-state i AppContext i stedet? Bytt ut nåværende bruk av toast med denne, og synliggjør feilmeldingen på et mer hensiktsmessig sted - som for eksempel under hele nytt barn samme partner-boksen?

Reply:
Problemet med å "plassere den" er at den kan opptre på svært mange steder og skje når brukerne har "scrollet" bort fra elementet som ble trykket på. Så tenker toast/modal-feil er mest nærliggende i så måte.